packages/hardhat-ledger/src/type-extensions.ts

```diff
@@ -2,16 +2,28 @@ import "hardhat/types/config";
 
 declare module "hardhat/types/config" {
   interface HardhatNetworkUserConfig {
-    ledgerAccounts?: string[];
```

Comment:
this should still be optional

Reply:
Agreed. I've also made `ledgerOptions` optional here.